activities/membership-growth/membership-proposal-template.md

```diff
@@ -0,0 +1,116 @@
+---
+type: Resource
+redirect_from:
+    - activities/membership-growth/membership-proposal-template.md
```

Comment:
This page hasn't been moved so doesn't need to redirect from anywhere. However, you could add paths here of files you've deleted that should be redirected to this page.

Reply:
I *hope* I fixed it correctly now! Added 2 removed files to the redirect.